lib/davinci_dtr_test_kit/profiles/questionnaire_response_group.rb

```diff
@@ -0,0 +1,66 @@
+require_relative 'questionnaire_response/questionnaire_response_patient_search'
+require_relative 'questionnaire_response/questionnaire_response_context_search'
+require_relative 'questionnaire_response/questionnaire_response_read'
+require_relative 'questionnaire_response/questionnaire_response_validation'
+require_relative 'questionnaire_response/questionnaire_response_create'
+require_relative 'questionnaire_response/questionnaire_response_update'
+
+module DaVinciDTRTestKit
+  class QuestionnaireResponseGroup < Inferno::TestGroup
+    title 'DTR QuestionnaireResponse Tests'
+    short_description 'Verify support for the server capabilities required by the DTR QuestionnaireResponse Profile'
+    description %(
+      # Background
+
+    The DTR QuestionnaireResponse sequences verifies that the system under test is able to create and update
+    QuestionnaireResponse instances and provide correct responses for QuestionnaireResponse queries. These queries must
+    return resources conforming to the [DTR QuestionnaireResponse Profile](http://hl7.org/fhir/us/davinci-dtr/STU2/StructureDefinition-dtr-questionnaireresponse.html).
+
+    # Testing Methodology
+    ## Searching
+    This test sequence will first perform each required search associated with this resource. This sequence will perform
+    searches with the following parameters:
+
+    * `patient`
+    * `context`
+
+    ### Search Parameters
+    The first search uses the patient(s) from the *Patient IDs* input. Subsequent searches will look for its parameter
```

Comment:
Update the description of the Patient IDs input to make clear that it is only getting used to search for QuestionnaireResponses.

Reply:
Addressed in [Update patient IDs input description](https://github.com/inferno-framework/davinci-dtr-test-kit/pull/35/commits/30e1d4f9b3bd153e9029146b2fa959712feb2b2f)